### DISCUSSION TITLE
How To Use Laravel Settings With Livewire?

### DISCUSSION THREAD

**Discussion by reasecret** [spatie/laravel-settings#152] [Q&A] [answered]
I'm using Livewire for my project on localhost and want to try laravel-settings. Documentation shows using with controllers. But I'm not using controllers, using Livewire components. And I don't know how to use settings with Livewire. I couldn't find anything about this.

Is there anyone could share some example for this?

**Comment by rubenvanassche** [comment, marked as answer]
You can inject settings anywhere you can inject something from the Laravel container. I think calling `resolve(GlobalSettings::class)` within your livewire component should work.

**Reply by reasecret** [reply]
Yes that worked. I can use settings with any Livewire component. Thanks...

**Reply by madurapa** [reply]
Now it should be `app(GlobalSettings::class)` instead of `resolve(GlobalSettings::class)`
